include/swift/AST/MacroDeclaration.h

```diff
@@ -97,8 +97,14 @@ enum class MacroIntroducedDeclNameKind {
   Prefixed,
   Suffixed,
   Arbitrary,
+
+  // NOTE: When adding a new name kind, also add it to
+  // `getAllMacroIntroducedDeclNameKinds`.
```

Comment:
I haven't looked at all the uses of `MacroIntroducedDeclNameKind`, could we have a `Last` equivalent rather than the vector?

Reply:
Discussed offline and couldn’t come up with a betters solution than the `std::vector`